src/test/java/SetTest.java

```diff
@@ -0,0 +1,48 @@
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("Set 내 중복없이 데이터 사이즈가 계산되는지 테스트")
+    void sizeTest() {
+        assertThat(numbers.size()).isEqualTo(3);
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = {1, 2, 3})
+    @DisplayName("Set 내 파라마티러 받은 값이 모두 포함되어있는지 테스트")
+    void containsTestWithValueSource(int input) {
+        assertThat(numbers.contains(input)).isTrue();
```

Comment:
특정 값을 가지고 있는지 검증하는 메서드를 사용해보면 어떨까요?
```suggestion
        assertThat(numbers).contains(input);
```